### PR TITLE
Polish changes:

### LINKED ISSUE #2359: [Dashboard v1][Tracker][Redesign] Landing Page v1
## General Idea
Create a landing page that provides all existing functionality while also starting to offer some basic help and information. This is a tracker issue. I will be adding these features!

## Features
Tentative list of features being added:
- Create a responsive and a11y enabled **sidebar**
  - Move links to a sidebar
  - Add link for Home
  - Add Kubeflow Logo
- Create a **Main View**
  - Either contains the page (**`Dashboard`** / **`Activity`**)
  - Or an Iframe view which loads all the help links!
- Contains a **_header toolbar_** (*in **Main View***)
  - Ability to swap between **Dashboard** and **Activity**

## Backend
Goals:
- Provide REST APIs to expose:
  - Namespaces
  - Build information
  - Any relevant metrics?

### Dashboard
- `paper-card` that contains action items to many of the common actions needed to be performed on the cluster
- Possibly a chart :)

### PR DESCRIPTION
**Pull request by avdaredevil**
## Features
- List box text was not visible in blue toolbar mode
- Colors adjusted for clearer viewing of namespace selector
- Made the message much easier to read and made it an empty state message

/assign @avdaredevil @prodonjs 
/review @prodonjs 
/priority p0

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2733)

### CODE REVIEW

File: components/centraldashboard/public/components/activity-view.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
 import '@polymer/iron-ajax/iron-ajax.js';
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-icons/iron-icons.js';
+import '@polymer/paper-card/paper-card.js';
```
Comment (prodonjs) [inline components/centraldashboard/public/components/activity-view.js]: Any reason this is added?

Comment (avdaredevil) [reply components/centraldashboard/public/components/activity-view.js]: It was added for a reason that is no longer required. Removed, good catch!

**Comment by avdaredevil** [comment]
Hey @prodonjs can you please test `gcr.io/kubeflow-images-public/centraldashboard:v20190318-v0.4.0-rc.1-237-gb0f6b6a0` it seems a reload on a custom route fails.

**Comment by prodonjs** [comment]
I didn't see any issues with the deployed image when trying to navigate directly to a deep-link (ie. https://kubeflow.endpoints.prodonjs-kubeflow-dev.cloud.goog/_/notebooks) or to a non-existent route (ie. https://kubeflow.endpoints.prodonjs-kubeflow-dev.cloud.goog/a/bad/route).

The presubmit failure is unrelated to the change here as our image is building correctly.

/lgtm
/approve

**Comment by prodonjs** [comment]
/assign @gaocegege 

Need owners approval for jsonnet file change.

**Comment by kunmingg** [comment]
@avdaredevil 
https://github.com/kubeflow/kubeflow/pull/2729 merged, can you rebase?

**Comment by avdaredevil** [comment]
Done, could you please re-lgtm?

**Pull request merged**